### DISCUSSION TITLE
Required procedure to install the PgAdmin to access the postgres data on Cqube

### DISCUSSION THREAD

**Discussion by vskmizoram** [Sunbird-cQube/community#856] [Q&A]
Need the procedure to install the PGAdmin   on Cqube

**Comment by bommalla** [comment]
@vskmizoram @RNarmatha 

There is no specific procedure. Please download the pgAdmin4 in your ubuntu server and enable the port number 5432 and try to login.Please find the attached screenshot for your reference.

![image](https://github.com/Sunbird-cQube/community/assets/86966482/b3b12641-ce34-47d7-84b6-160d1ef50153)

**Reply by vskmizoram** [reply]
Thanx... I had successfully installed the PgAdmin, what is the password of the Postgres installed through cQube. It is a docker file, how can we get access through PgAdmin to the cQube data tables

**Comment by bommalla** [comment]
@vskmizoram @RNarmatha 

user_name : cqube_user
password: cQube@123
